src/models/Device.ts

```diff
@@ -203,6 +203,8 @@ export class NrfDevice {
 				}
 			}
 		}
+		// @ts-ignore, Adding in a flag that we check for when we fetch the device in nrfDevice.ts to populate shadow.
+		shadow.state.reported.simulator = true;
```

Comment:
This is a good feature just for users (or any other tools/scripts) to easily know it's a device created by the simulator. Independent of anything the simulator itself uses it for.
Could be useful to the FE too, maybe marking these devices in a different color or something. The `nrfsim-...` naming convention can be replaced by the user, good to have something different to indicate simulated.